src/uct/tcp/tcp_cm.c

```diff
@@ -438,6 +442,9 @@ static void uct_tcp_cm_handle_conn_ack(uct_tcp_ep_t *ep)
     if (ep->conn_state != UCT_TCP_EP_CONN_STATE_CONNECTED) {
         uct_tcp_cm_change_conn_state(ep, UCT_TCP_EP_CONN_STATE_CONNECTED);
     }
+
+    /* Progress possibly pending TX operations */
+    uct_tcp_ep_pending_queue_dispatch(ep);
```

Comment:
maybe better call it in a single place, when connection state moves to CONNECTED?

Reply:
done